test_functions.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+import utility
+
+# testing with already existing uuid
+def test_valid_uuid():
+    with pytest.raises(Exception) as e_info:
+        utility.create_uuid_container_in_azure('u-1234abcd5454efef8842ll3fsc')
+
+# testing with shorter length uuid
+def test_valid_uuid1():
+    with pytest.raises(Exception) as e_info:
+        utility.create_uuid_container_in_azure('u-1234abcd5454efef8842ll3fs')
+
+# testing with special case in uuid
+def test_valid_uuid2():
+    with pytest.raises(Exception) as e_info:
+        utility.create_uuid_container_in_azure('u-1234abcd5454efef8842ll3!ss')
+
+# should be not passed. but how?
+def test_valid_uuid4():
+    with pytest.raises(Exception) as e_info:
+        utility.create_uuid_container_in_azure('u-1234abcd5454efef8842llaaaa')
```

Comment:
`test_valid_uuid1`, `test_valid_uuid2`, `test_valid_uuid4` can be combined into `test_create_uuid_container_in_azure`

**you can either use https://docs.pytest.org/en/latest/parametrize.html (preferred)**
or
you can use a list of dict or object
```python
inputs = [{"input": "1234abcd5454efef8842ll3fsc'", "is_err_expected": False...},
    {"input": "1234abcd5454efef8842ll3fsc'", "is_err_expected": False...}, ...
    ]
```
you can have as much dict containing test cases

Reply:
try your best to test all the functions like a function called `test_get_file_type` etc

Reply:
If it is okay, I will create a new pull request that has changes regarding testing.